### DISCUSSION TITLE
Can the `github-tags dataSourceTemplate` return the file digest not just the commit digest?

### DISCUSSION THREAD

**Discussion by molinamelendezj** [renovatebot/renovate#29632] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab and renovateVersion 37.391.0

### Please tell us more about your question or problem

I've successfully used the `github-release-attachments` datasourceTemplate to get the file digest (256 checksum) for custom release attachments that use the `https://github.com/*/*/releases/download` link, but I cannot get it to work for the github release source archive which looks like `https://github.com/*/*/archive/refs/tags/*.tar.gz`

I've tried using the `github-tags dataSourceTemplate` but it uses the tag commit as the digest. Is there a way to get this archive file checksum, like `github-release-attachments` does for custom release attachments. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
No, this is not possible
